app/src/main/java/com/xffffff/wellfed/MainActivity.java

```diff
@@ -75,6 +75,9 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
             }
         });
 
+        // fixes crashes related to nested fragments
+        viewPager.setOffscreenPageLimit(3);
```

Comment:
I am not sure, what it does can you explain a bit other than that i am approving it